lld/COFF/DriverUtils.cpp

```diff
@@ -607,6 +608,12 @@ Export LinkerDriver::parseExport(StringRef arg) {
       e.isPrivate = true;
       continue;
     }
+    if (tok.equals_insensitive("exportas")) {
+      if (rest.empty() || rest.contains(','))
```

Comment:
I wonder if we should improve the error message here - it might not be obvious that `EXPORTAS` must be the last item if its present.

Reply:
I improved it in the new version.